ansys/dpf/core/plotter.py

```diff
@@ -74,6 +74,37 @@ def add_field(self, field, meshed_region=None, **kwargs):
         # plot
         self._plotter.add_mesh(meshed_region.grid, scalars=overall_data, **kwargs)
 
+        if show_max or show_min:
+            # Get Min-Max for the field
+            min_max = core.operators.min_max.min_max()
+            min_max.inputs.connect(field)
+
+        # Add Min and Max Labels
+        labels = []
+        grid_points = []
+        if show_max:
+            max_value = min_max.outputs.field_max().data[0]
+            # Get max value index
+            max_value_index = np.where(overall_data == max_value)[0][0]
+            # Get node ID at max value
+            node_id_at_max = min_max.outputs.field_max().scoping.ids[0]
+            labels.append(f"Max: {min_value:.2f}, NodeID: {node_id_at_min}")
+            grid_points.append(meshed_region.grid.points[max_value_index])
+
+        if show_min:
+            min_value = min_max.outputs.field_min().data[0]
+            # Get max value index
+            min_value_index = np.where(overall_data == min_value)[0][0]
```

Comment:
@anslpa , @cbellot000 don't we have a map from node ids to grid indices? this will avoid inconsistencies when several nodes are on the min or max. @ayush-kumar-423

Reply:
I couldn't find any, if there is one, I agree it makes sense to use that.

Reply:
Doesn't the output from `min_max.outputs.field_min()` include a scoping?

Reply:
@akaszynski There is a `meshed_region` attribute, however `min_value.meshed_region.grid` throw the following exception,
Ideally `min_value.meshed_region.grid.points` should return all the points scoped for this value?

`DPFServerNullObject: object is null in the dataBase`

Reply:
@akaszynski `min_max.outputs.field_min()` includes a scoping that @ayush-kumar-423 used here: `node_id_at_min = min_max.outputs.field_min().scoping.ids[0]`.
@ayush-kumar-423 you can't do that: `min_value.meshed_region.grid` because the support of a min (or max) field coming from the operator "min_max" is not a mesh (for consistency with the support out of "min_max_fc" which is a timefreq support). But I don't think that you need this: `min_value.meshed_region.grid`. You already have a meshed region and you can use it like that:
```
min = min_max.outputs.field_min()
node_id_at_min = min.scoping.id(0)
labels.append(f"Min: {min.data[0]:.2f}\nNodeID: {node_id_at_min}"
grid_points.append(meshed_region.grid.points[meshed_region.nodes.scoping.index(node_id_at_min)])
```
FYI if the field in input of this method is "Elemental" your code won't work.

Reply:
Thanks  @cbellot000!